.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
   plugins: ["react", "react-hooks", "prettier", "jest", "json"],
   rules: {
     "react/prop-types": [2, { ignore: ["children"] }],
+    "jest/no-hooks": "off",
```

Comment:
This was actually quite useful 👀

Reply:
Was it? I don't see much sense in limiting the `beforeEach` setup functions 🤔